entry_types/scrolled/package/src/frontend/__stories__/playerControls-stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import {ControlBar} from 'pageflow-scrolled/frontend';
+
+const stories = storiesOf('Frontend/Player Controls', module);
+
+function addControlbarStory(name, type, style) {
+  let playerAreaColor = (style === 'white' ? 'black' : 'white');
+  stories.add(
+    name,
+    () => {
+      return (
+        <div style={{fontFamily: 'Source Sans Pro, sans-serif'}}>
+          <div style={{background: playerAreaColor, width: '100%', height: '150px'}}></div>
+          <ControlBar type={type} style={style}/>
+        </div>
+      );
+    },
+    {
+      percy: {skip: true}
```

Comment:
Those could make sense also in Percy, couldn't they?

Reply:
true, the controls are now added to Percy